README.md

```diff
@@ -56,8 +56,8 @@ be thoroughly reviewed.
 
 ### What changes we'll (most likely) not accept
 
-Most changes might be UI-related, and we think it's more or less a waste of time. If something design-related gets changed,
-ideally the whole app must follow the new style too, and that's hard to accomplish, especially for mods. That's why:
+Most changes might be UI-related, and we think it is more or less waste of time. If a something design-related gets changed,
```

Comment:
``If a``
Are you sure about your grammar bruh.

Reply:
also it should be: `more or less a waste of time`